client/src/redux/actions/scansActions.js

```diff
@@ -1,22 +1,35 @@
 import * as types from '../constants/scansConstants';
 import scansApi from '../../services/scansApi';
 
-const loadScansSuccess = data => ({
+const scansError = (bool, message) => ({
+  type: types.LOAD_SCANS_ERROR,
+  error: bool,
+  message: message
+});
+
+const scansLoading = bool => ({
+  type: types.LOAD_SCANS_LOADING,
+  loading: bool
+});
+
+const scansSuccess = data => ({
   type: types.LOAD_SCANS_SUCCESS,
   data
 });
 
 const getScans = () => {
   return function(dispatch) {
+    dispatch(scansLoading(true));
     return scansApi
-      .getScans()
+      .getSources()
```

Comment:
Should still be `getScans()`

Reply:
good catch!

Reply:
looks like creds is the same... fixing that one too.